src/lib/loaders/loaders_with_authentication/causality.ts

```diff
@@ -0,0 +1,27 @@
+import factories from "../api"
+import config from "config"
+
+const { EXCHANGE_APP_ID } = config
+
+// TODO: Maybe move causality JWT code from causality/link.ts into here.
+// other wise delet this
```

Comment:
this file is just copied completely so we may want to delete it.

Reply:
The exchange version of this file is used for setting up the token auth, which comes from gravity. Causality's JWT is made in the causality_jwt.ts resolver- `causalityJwt` is its own `String` field already used for live auction websocket connections. I extracted the jwt() call to a separate export in that file for now, but maybe we could factor it out into a causalityTokenLoader here and use that in both places.

Or, maybe we should just allow the repetition. We make a causality jwt by saying `meLoader.then( ({id}) => causalityJwt({userId: id, role, bidderId, saleId}) )` where none of these params but `role` absolutely required.